### PR TITLE
Github API GET verb testing

### PR DESCRIPTION
**Pull request by manuelq12**
Project with new file that test the Github API with GET verbs.

### CODE REVIEW

File: test/GithubApi.Repositories.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+describe('Github Api Test', () => {
+  describe('Users & Repositories', () => {
+    before(async () => {
+      userInfo = await agent.get(`${urlBase}/users/${githubUserName}`)
+        .auth('token', process.env.ACCESS_TOKEN)
+        .set('User-Agent', 'agent');
+    });
+
+    it('Via GET verb obtain user info', async () => {
```
Comment (aperdomob) [inline test/GithubApi.Repositories.test.js:25]: async is not needed

File: test/GithubApi.Repositories.test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+
+const { expect } = chai;
+
+let userInfo = null;
```
Comment (aperdomob) [inline test/GithubApi.Repositories.test.js:11]: define this variable into the describes

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: test/GithubApi.Repositories.test.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        .set('User-Agent', 'agent');
+
+      const repository = response.body.find((element) => element.name === repositoryName);
+      expect(response.status).to.equal(statusCode.OK);
+      expect(repository.full_name).to.equal(`${githubUserName}/${repositoryName}`);
+      expect(repository.private).to.equal(false);
+      expect(repository.description).to.equal('An awesome html report for Jasmine');
+    });
+
+    it('Via GET verb download jasmine-awesome-report repository', async () => {
```
Comment (saradrada) [inline test/GithubApi.Repositories.test.js:44]: there should be a describe with the action you're testing and then an it with the assertions

File: test/GithubApi.Repositories.test.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      const response = await agent.get(`${urlBase}/repos/${githubUserName}/${repositoryName}/zipball`)
+        .auth('token', process.env.ACCESS_TOKEN)
+        .set('User-Agent', 'agent');
+
+      expect(response.status).to.equal(statusCode.OK);
+      expect(response.header['content-type']).to.equal('application/zip');
+    });
+
+    it('Via GET verb obtain jasmine-awesome-report README.md info', async () => {
```
Comment (saradrada) [inline test/GithubApi.Repositories.test.js:53]: same here, describe with the action and it with the assertions

File: test/GithubApi.Repositories.test.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        .set('User-Agent', 'agent');
+
+      const readme = response.body.find((element) => element.name === 'README.md');
+      expect(readme).to.containSubset({
+        path: 'README.md',
+        sha: 'b9900ca9b34077fe6a8f2aaa37a173824fa9751d'
+      });
+    });
+
+    it('Via GET verb download jasmine-awesome-report README.md and check MD5 hash', async () => {
```
Comment (saradrada) [inline test/GithubApi.Repositories.test.js:65]: same here, describe with the action and it with the assertions

File: test/GithubApi.Repositories.test.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    });
+
+    it('Via GET verb obtain user info', async () => {
+      expect(userInfo.status).to.equal(statusCode.OK);
+      expect(userInfo.body.name).equal('Alejandro Perdomo');
+      expect(userInfo.body.company).equal('PSL');
+      expect(userInfo.body.location).equal('Colombia');
+    });
+
+    it('Via GET verb obtain user repositories info', async () => {
```
Comment (saradrada) [inline test/GithubApi.Repositories.test.js:32]: there should be a describe with the action you're testing and then an it with the assertions

File: test/GithubApi.Repositories.test.js (hunk 2 above)
Comment (saradrada) [inline test/GithubApi.Repositories.test.js:11]: this variable shouldn't be global

**Review by saradrada — CHANGES_REQUESTED** [review]
Manu please improve the structure of the tests by separating the preparation of the data, the action you're testing and the assertions

File: .eslintrc.yml [hunk 7]
```diff
@@ -10,4 +10,3 @@ extends:
 rules:
   "comma-dangle": ["error", "never"]
   linebreak-style: ["error", "unix"]
-
```
Comment (saradrada) [inline .eslintrc.yml:13]: please add a new line at the EOF

**Review by saradrada — CHANGES_REQUESTED** [review]

**Review by saradrada — APPROVED** [review]

**Pull request merged**